src.gen/@amzn/codewhisperer-streaming/dist-es/runtimeConfig.shared.js

```diff
@@ -0,0 +1,18 @@
+import { defaultRegionInfoProvider } from "./endpoints";
```

Comment:
same question about `/dist-es/`. if we are committing dist folders, why do we have the typescript project?

if we can avoid the typescript project, that's great. but if we can't, then there is no reason to commit its build results?

Reply:
This was an oversight, fixed now.